include/zephyr/arch/x86/x86_acpi_osal.h

```diff
@@ -26,4 +26,13 @@ static inline void *acpi_rsdp_get(void)
 	return bios_acpi_rsdp_get();
 }
 #endif /* CONFIG_X86_EFI */
+
+static inline uint64_t acpi_timer_get(void)
```

Comment:
`acpi` -> `apic_tsc` ?

Reply:
that is platform specific and this api is generic (currently only for Intel platform)

Reply:
So you are using APIC TSC, similar to
https://github.com/zephyrproject-rtos/zephyr/blob/97fab8371651ba4c4b9e5ee91759c2435a135306/drivers/timer/apic_tsc.c#L30-L36

There is also ACPI timer: https://wiki.osdev.org/ACPI_Timer Looks like ACPI timer is different from what you call acpi_timer.

Reply:
This is just ACPICA OS service layer implementation for AcpiOsGetTimer interface and for Intel platforms currently use TSC.